src/router_core/transfer.c

```diff
@@ -671,7 +708,8 @@ static void qdr_send_to_CT(qdr_core_t *core, qdr_action_t *action, bool discard)
 static void qdr_update_delivery_CT(qdr_core_t *core, qdr_action_t *action, bool discard)
 {
     qdr_delivery_t *dlv        = action->args.delivery.delivery;
-    qdr_delivery_t *peer       = dlv->peer;
+    qdr_delivery_start_peer_CT(dlv);
```

Comment:
Now that I'm looking at it, a better API might be: qdr_delivery_t qdr_delivery_first_peer_CT(dlv) instead of start_peer.